### PR TITLE
Make LED buttons controllable

### PR DESCRIPTION
**Pull request by brefra**
This PR makes the LED's of buttons controllable so it possible to “link” a button to an external device (I.e. when using Home Assistant linking to an entity could be done with some smart automation rules) and allow the LED to indicates if the external device is actually on / off.

I've added `light_is_buttonled()` to allow distinction between regular controllable lights and LEDs of the buttons. This allows the Home Assistant integration to default mark all LEDs as _disabled_ "light" entities so only the usable LED's can be manually enabled.

### CODE REVIEW

**Comment by Cereal2nd** [comment]
why did you mark these as light entity and not switch entity?

these are not dimable, so we should mark them as a switch entity in hass

**Comment by brefra** [comment]
That is indeed a point of discussion if we want to keep it simple.

The reason I have chosen a light entity is that HASS also support [flashing](https://www.home-assistant.io/integrations/light/) for light entities.
So you're able to have the LED flashing by appending `flash: long` or `flash: short` to the `light.turn_on` service call. That's the reason i've added [this code](https://github.com/thomasdelaet/python-velbus/pull/38/files#diff-10ad0b8784d64ae2ed2a9f2a01e40297R145-R168)

So basically it's not only turning on/off but also flashing ;-)

A "how do I setup the linking (automation) and flashing in HASS?" is definitely something to be explained at the Velbus integration documentation. I'll add it together when making the required PR voor HASS, which i've ready but is waiting (and depending) for this PR.

**Comment by Cereal2nd** [comment]
Ok for the light component

if we want to use the flasing thing we will need to support a list of features together with the entity type otherwise hass does not know what types it needs to enable in its entity.

**Comment by brefra** [comment]
That is the reason why I've added `light_is_buttonled()` as mentioned in the PR comment. This allows the HASS integration code to enable the flash feature for the LED buttons too.
In my opinion this library should not have any list of HASS features as that belongs at the HASS side.

If you are interested I'll push the HASS integration code (an updated light.py) to a separate branch at my github HASS fork so you can have a look at it (or even test it yourself as an custom component together with my fork of this lib as dep, which I'm running myself already)

**Comment by Cereal2nd** [comment]
ok, cool

please push the code code somewhere so i can play with it

**Comment by brefra** [comment]
HASS code is available [here](https://github.com/brefra/home-assistant/tree/velbus-led-control/homeassistant/components/velbus)
